pkgs/top-level/all-packages.nix

```diff
@@ -3145,6 +3145,8 @@ in
 
   dev86 = callPackage ../development/compilers/dev86 { };
 
+  diagrams = with python3.pkgs; toPythonApplication diagrams;
```

Comment:
@jonringer I just wonder, is it good practice to duplicate python packages listed in python-packages.nix that have some cli tools to all-packages.nix?

Reply:
In the case of OpenRazer, I think it was appropriate due to the nature of the package, but I am also curious about best practices, especially in this case.

Reply:
This makes it so you can install `diagrams` and have the executable, as well as use `python3Packages.diagrams` if you want to import the library.

Read up on it here https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#topythonapplication-function.

Reply:
I reverted this because, for this diagrams library, it does not make sense as an executable.